### DISCUSSION TITLE
Arbitrary Semver Increase When Dependencies are Updated in PR? Possible?

### DISCUSSION THREAD

**Discussion by ekristen** [renovatebot/renovate#12343] [Request Help] [answered]
Is there a way to have renovate bot increase a semver value of a non-dependency when it is updating dependencies in a pull request?

For example, with helm, it might detect a new dependency update and open a PR, but the helm chart.yaml has it's own `version` that should be increased anytime a dependency is changed. 

Another example would be node and package.json, with a PR to update a dependency can the `version` field in the package.json be automatically increased?

**Comment by viceice** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#bumpversion
